backend/server.py

```
@@ -366,75 +529,111 @@ def generate_gform():
+    try:
+        input_text = process_input_text(input_text, use_mediawiki)
+        output = qg.generate(
+            article=input_text,
+            num_questions=input_questions,
+            answer_style="sentences",
+        )
+        for item in output:
+            item["question"] = make_question_harder(item["question"])
+        logger.info("Generated %d hard short questions", len(output))
+        return jsonify({"output": output})
+    except Exception as e:
```

Ruff (0.14.14)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)